Test App crash at startup

After a restart the app started again.

Environment: test

```
[ERROR] TIME: 2024-03-15T07:19:07.198383Z r: Aborting after panic in task info: panicked at /Users/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/state-0.5.3/src/storage.rs:162:14:
storage::get(): called get() before set()
```

full logs

```
HybridOutputLogger initialized. Will print into console and '/data/user/0/finance.get10101.app.demo/files/app_log.log'
[INFO] TIME: 2024-03-15T07:19:06.786887Z r: Initialized logger 
[INFO] TIME: 2024-03-15T07:19:06.810418Z User granted permission: AuthorizationStatus.authorized
[INFO] TIME: 2024-03-15T07:19:06.810938Z Built on commit: 6cfe50227d41738d94d8c13c280b13bd1e5b3054
[INFO] TIME: 2024-03-15T07:19:06.810963Z Built on branch: main
[INFO] TIME: 2024-03-15T07:19:06.811697Z App data will be stored in: /data/user/0/finance.get10101.app.demo/files
[INFO] TIME: 2024-03-15T07:19:06.811726Z Seed data will be stored in: /data/user/0/finance.get10101.app.demo/files
[INFO] TIME: 2024-03-15T07:19:06.813534Z App has already data in the seed dir. For compatibility reasons we will not switch to the new app dir.
[DEBUG] TIME: 2024-03-15T07:19:06.832541Z r: Parsing config from flutter config: Config { coordinator_pubkey: "03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802", electrs_endpoint: "http://34.32.0.52:3000", host: "34.32.0.52", p2p_port: 9045, http_port: 80, network: "regtest", oracle_endpoint: "http://34.32.0.52:8081", oracle_pubkey: "5d12d79f575b8d99523797c46441c0549eb0defb6195fe8a080000cbe3ab3859", health_check_interval_secs: 10 }
[INFO] TIME: 2024-03-15T07:19:06.833141Z Build number: 3596
[INFO] TIME: 2024-03-15T07:19:06.833175Z Build version: 2.0.4
[INFO] TIME: 2024-03-15T07:19:06.833189Z Network: regtest
[INFO] TIME: 2024-03-15T07:19:06.833194Z Electrs endpoint: http://34.32.0.52:3000
[INFO] TIME: 2024-03-15T07:19:06.833202Z Coordinator: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@34.32.0.52:9045
[INFO] TIME: 2024-03-15T07:19:06.833210Z Oracle endpoint: http://34.32.0.52:8081
[INFO] TIME: 2024-03-15T07:19:06.833214Z Oracle PK: 5d12d79f575b8d99523797c46441c0549eb0defb6195fe8a080000cbe3ab3859
[DEBUG] TIME: 2024-03-15T07:19:06.833420Z r: Subscribing flutter to event hub 
[DEBUG] TIME: 2024-03-15T07:19:06.833534Z Scanning for seed file in: /data/user/0/finance.get10101.app.demo/files/regtest/seed
[INFO] TIME: 2024-03-15T07:19:06.928731Z Coordinator version: 2.0.4
[INFO] TIME: 2024-03-15T07:19:06.928826Z Client is up to date: 2.0.4
[DEBUG] TIME: 2024-03-15T07:19:06.928931Z r: Database migration run - db initialized 
[DEBUG] TIME: 2024-03-15T07:19:06.928987Z r: Opening read-only backup connection 
[DEBUG] TIME: 2024-03-15T07:19:06.933183Z found no subscribers, skipping event
[INFO] TIME: 2024-03-15T07:19:06.958486Z r: Created backup dir at /data/user/0/finance.get10101.app.demo/files/regtest/backup 
[INFO] TIME: 2024-03-15T07:19:07.114434Z r: Initialized 10101 storage! 
[WARN] TIME: 2024-03-15T07:19:07.114660Z r: Initializing fee rate cache with default values. defaults: [(MempoolMinimum, 1000), (Background, 2000), (Normal, 3000), (HighPriority, 4000)]
[INFO] TIME: 2024-03-15T07:19:07.118166Z r: Creating on-chain wallet network: Regtest
[INFO] TIME: 2024-03-15T07:19:07.185328Z r: Found channel manager data. Recovering from stored state 
[INFO] TIME: 2024-03-15T07:19:07.190360Z r: Listening on 0.0.0.0:33333 
[INFO] TIME: 2024-03-15T07:19:07.190934Z r: Starting background processor 
[INFO] TIME: 2024-03-15T07:19:07.192249Z r: Lightning node started with node ID tcp://029b9001be0c6abfd6a1e0e6c492e2bcb2700c8e93a53befa3a6fa9ebcf391c05b@0.0.0.0:33333 
[DEBUG] TIME: 2024-03-15T07:19:07.192733Z r: Got all transactions transactions: []
[DEBUG] TIME: 2024-03-15T07:19:07.192830Z r: Syncing 0 shadow transactions 
[DEBUG] TIME: 2024-03-15T07:19:07.192920Z r: Setting up connection peer: tcp://03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@34.32.0.52:9045
[DEBUG] TIME: 2024-03-15T07:19:07.192965Z r: Connecting over TCP 
[DEBUG] TIME: 2024-03-15T07:19:07.194734Z found no subscribers, skipping event
[DEBUG] TIME: 2024-03-15T07:19:07.195025Z r: Got all spendable outputs outputs: []
[ERROR] TIME: 2024-03-15T07:19:07.198383Z r: Aborting after panic in task info: panicked at /Users/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/state-0.5.3/src/storage.rs:162:14:
storage::get(): called get() before set()
[DEBUG] TIME: 2024-03-15T07:19:07.216Z r: Connecting to orderbook API 
[DEBUG] TIME: 2024-03-15T07:19:07.220183Z r: Pruning expired orders 
[INFO] TIME: 2024-03-15T07:19:07.223850Z Backend started
[DEBUG] TIME: 2024-03-15T07:19:07.249059Z r: Connection setup completed peer: tcp://03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@34.32.0.52:9045
[DEBUG] TIME: 2024-03-15T07:19:07.249330Z r: Waiting to confirm established connection peer: tcp://03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@34.32.0.52:9045
[DEBUG] TIME: 2024-03-15T07:19:07.308759Z r: lightning::ln::peer_handler: Finished noise handshake for connection with 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802 
[DEBUG] TIME: 2024-03-15T07:19:07.311963Z r: Response status code 200 OK 
[DEBUG] TIME: 2024-03-15T07:19:07.312120Z r: Successfully uploaded backup of dlc/03/636861696e5f6d6f6e69746f72. 
[DEBUG] TIME: 2024-03-15T07:19:07.319617Z r: tungstenite::handshake::client: Client handshake done. 
[INFO] TIME: 2024-03-15T07:19:07.319786Z r: Connected to orderbook realtime API 
[DEBUG] TIME: 2024-03-15T07:19:07.361548Z r: Response status code 200 OK 
[DEBUG] TIME: 2024-03-15T07:19:07.361708Z r: Successfully uploaded backup of dlc/03/636861696e5f6d6f6e69746f72. 
[INFO] TIME: 2024-03-15T07:19:07.361764Z r: lightning::ln::peer_handler: Received peer Init message from 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802: DataLossProtect: required, InitialRoutingSync: not supported, UpfrontShutdownScript: supported, GossipQueries: supported, VariableLengthOnion: required, StaticRemoteKey: required, PaymentSecret: required, BasicMPP: supported, Wumbo: supported, AnchorsNonzeroFeeHtlcTx: not supported, AnchorsZeroFeeHtlcTx: not supported, ShutdownAnySegwit: supported, Taproot: not supported, OnionMessages: not supported, ChannelType: supported, SCIDPrivacy: supported, ZeroConf: supported, unknown flags: none 
[DEBUG] TIME: 2024-03-15T07:19:07.361959Z r: lightning::ln::channelmanager: Generating channel_reestablish events for 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802 
[INFO] TIME: 2024-03-15T07:19:07.362074Z r: Peer connected! inbound: false,their_node_id: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802
[DEBUG] TIME: 2024-03-15T07:19:07.364252Z r: Sending last dlc message last_outbound_serialized_dlc_message.message_type: CollaborativeCloseOffer,peer: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802
[DEBUG] TIME: 2024-03-15T07:19:07.373310Z r: Fetched new polls new_polls: 1
[DEBUG] TIME: 2024-03-15T07:19:07.373447Z r: Polls to answer unanswered_polls: 1
[DEBUG] TIME: 2024-03-15T07:19:07.373487Z r: Unanswered polls poll_id: 1
[DEBUG] TIME: 2024-03-15T07:19:07.458378Z r: lightning::chain::chainmonitor: New best block 6bd81133cebb31ab679036f330a863a0ccf1abc6946e30da9ea8f74df2822a2a at height 251585 provided via best_block_updated 
[INFO] TIME: 2024-03-15T07:19:07.602487Z r: Processing message from: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802,kind: ChannelSettleConfirm
[DEBUG] TIME: 2024-03-15T07:19:07.602717Z r: Received channel message from: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802
[DEBUG] TIME: 2024-03-15T07:19:07.603693Z r: Received message that has already been processed, skipping. kind: ChannelSettleConfirm,node_id: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802
HybridOutputLogger initialized. Will print into console and '/data/user/0/finance.get10101.app.demo/files/app_log.log'
[INFO] TIME: 2024-03-15T07:19:10.389049Z r: Initialized logger 
[INFO] TIME: 2024-03-15T07:19:10.432428Z User granted permission: AuthorizationStatus.authorized
[INFO] TIME: 2024-03-15T07:19:10.432929Z Built on commit: 6cfe50227d41738d94d8c13c280b13bd1e5b3054
[INFO] TIME: 2024-03-15T07:19:10.432962Z Built on branch: main
[INFO] TIME: 2024-03-15T07:19:10.434225Z App data will be stored in: /data/user/0/finance.get10101.app.demo/files
[INFO] TIME: 2024-03-15T07:19:10.434249Z Seed data will be stored in: /data/user/0/finance.get10101.app.demo/files
[INFO] TIME: 2024-03-15T07:19:10.435469Z App has already data in the seed dir. For compatibility reasons we will not switch to the new app dir.
[DEBUG] TIME: 2024-03-15T07:19:10.443554Z r: Parsing config from flutter config: Config { coordinator_pubkey: "03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802", electrs_endpoint: "http://34.32.0.52:3000", host: "34.32.0.52", p2p_port: 9045, http_port: 80, network: "regtest", oracle_endpoint: "http://34.32.0.52:8081", oracle_pubkey: "5d12d79f575b8d99523797c46441c0549eb0defb6195fe8a080000cbe3ab3859", health_check_interval_secs: 10 }
[DEBUG] TIME: 2024-03-15T07:19:10.443799Z r: Subscribing flutter to event hub 
[INFO] TIME: 2024-03-15T07:19:10.443931Z Build number: 3596
[INFO] TIME: 2024-03-15T07:19:10.443957Z Build version: 2.0.4
[INFO] TIME: 2024-03-15T07:19:10.443964Z Network: regtest
[INFO] TIME: 2024-03-15T07:19:10.443977Z Electrs endpoint: http://34.32.0.52:3000
[INFO] TIME: 2024-03-15T07:19:10.443983Z Coordinator: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@34.32.0.52:9045
[INFO] TIME: 2024-03-15T07:19:10.444003Z Oracle endpoint: http://34.32.0.52:8081
[INFO] TIME: 2024-03-15T07:19:10.444014Z Oracle PK: 5d12d79f575b8d99523797c46441c0549eb0defb6195fe8a080000cbe3ab3859
[DEBUG] TIME: 2024-03-15T07:19:10.444083Z Scanning for seed file in: /data/user/0/finance.get10101.app.demo/files/regtest/seed
[DEBUG] TIME: 2024-03-15T07:19:10.460238Z r: Database migration run - db initialized 
[DEBUG] TIME: 2024-03-15T07:19:10.461029Z r: Opening read-only backup connection 
[DEBUG] TIME: 2024-03-15T07:19:10.465399Z found no subscribers, skipping event
[INFO] TIME: 2024-03-15T07:19:10.465596Z r: Created backup dir at /data/user/0/finance.get10101.app.demo/files/regtest/backup 
[INFO] TIME: 2024-03-15T07:19:10.529483Z Coordinator version: 2.0.4
[INFO] TIME: 2024-03-15T07:19:10.529580Z Client is up to date: 2.0.4
[INFO] TIME: 2024-03-15T07:19:10.547185Z r: Initialized 10101 storage! 
[WARN] TIME: 2024-03-15T07:19:10.548347Z r: Initializing fee rate cache with default values. defaults: [(MempoolMinimum, 1000), (Background, 2000), (Normal, 3000), (HighPriority, 4000)]
[INFO] TIME: 2024-03-15T07:19:10.556727Z r: Creating on-chain wallet network: Regtest
[INFO] TIME: 2024-03-15T07:19:10.612597Z r: Found channel manager data. Recovering from stored state 
[INFO] TIME: 2024-03-15T07:19:10.613867Z r: Listening on 0.0.0.0:35311 
[INFO] TIME: 2024-03-15T07:19:10.614127Z r: Starting background processor 
[DEBUG] TIME: 2024-03-15T07:19:10.618184Z r: Got all transactions transactions: []
[DEBUG] TIME: 2024-03-15T07:19:10.618341Z r: Syncing 0 shadow transactions 
[DEBUG] TIME: 2024-03-15T07:19:10.622767Z r: Got all spendable outputs outputs: []
[INFO] TIME: 2024-03-15T07:19:10.623732Z r: Lightning node started with node ID tcp://029b9001be0c6abfd6a1e0e6c492e2bcb2700c8e93a53befa3a6fa9ebcf391c05b@0.0.0.0:35311 
[ERROR] TIME: 2024-03-15T07:19:10.624887Z r: Aborting after panic in task info: panicked at /Users/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/state-0.5.3/src/storage.rs:162:14:
storage::get(): called get() before set()
[DEBUG] TIME: 2024-03-15T07:19:10.625123Z found no subscribers, skipping event
[DEBUG] TIME: 2024-03-15T07:19:10.642345Z r: Setting up connection peer: tcp://03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@34.32.0.52:9045
[DEBUG] TIME: 2024-03-15T07:19:10.642476Z r: Connecting over TCP 
[DEBUG] TIME: 2024-03-15T07:19:10.650728Z r: Connecting to orderbook API 
[DEBUG] TIME: 2024-03-15T07:19:10.650810Z r: Pruning expired orders 
[INFO] TIME: 2024-03-15T07:19:10.651546Z Backend started
[DEBUG] TIME: 2024-03-15T07:19:10.710735Z r: Connection setup completed peer: tcp://03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@34.32.0.52:9045
[DEBUG] TIME: 2024-03-15T07:19:10.710918Z r: Waiting to confirm established connection peer: tcp://03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@34.32.0.52:9045
[DEBUG] TIME: 2024-03-15T07:19:10.718619Z r: Response status code 200 OK 
[DEBUG] TIME: 2024-03-15T07:19:10.718701Z r: Successfully uploaded backup of dlc/03/636861696e5f6d6f6e69746f72. 
[DEBUG] TIME: 2024-03-15T07:19:10.738827Z r: tungstenite::handshake::client: Client handshake done. 
[INFO] TIME: 2024-03-15T07:19:10.738970Z r: Connected to orderbook realtime API 
[DEBUG] TIME: 2024-03-15T07:19:10.741085Z r: lightning::ln::peer_handler: Finished noise handshake for connection with 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802 
[DEBUG] TIME: 2024-03-15T07:19:10.742158Z r: Response status code 200 OK 
[DEBUG] TIME: 2024-03-15T07:19:10.742289Z r: Successfully uploaded backup of dlc/03/636861696e5f6d6f6e69746f72. 
[DEBUG] TIME: 2024-03-15T07:19:10.854003Z r: lightning::chain::chainmonitor: New best block 6bd81133cebb31ab679036f330a863a0ccf1abc6946e30da9ea8f74df2822a2a at height 251585 provided via best_block_updated 
```

--- a/mobile/native/src/ln_dlc/mod.rs
+++ b/mobile/native/src/ln_dlc/mod.rs
@@ -311,7 +311,9 @@ pub fn run(runtime: &Runtime) -> Result<()> {
 
         let event_handler = AppEventHandler::new(node.clone(), Some(event_sender));
         let _running = node.start(event_handler, true)?;
+
         let node = Arc::new(Node::new(node, _running));
+        state::set_node(node.clone());
 
         let dlc_handler = DlcHandler::new(node.clone());
         runtime.spawn(async move {
@@ -384,8 +386,6 @@ pub fn run(runtime: &Runtime) -> Result<()> {
                 tokio::time::sleep(CHECK_OPEN_ORDERS_INTERVAL).await;
             }
         });
-
-        state::set_node(node.clone());
 
         event::publish(&EventInternal::Init("10101 is ready.".to_string()));
 

